### ISSUE TITLE
Software destroys font-cache and crashes system

### ISSUE THREAD

**Issue by mr-who2012** [4gray/iptvnator#137]
**Describe the bug**
After opening the playlist-open dialog, all characters in the folder-view-window are replaced with empty boxes. The desktop-manager can't start any apps, afterwards. After reboot of system, the desktop-manager doesn't start-up anymore.  This did happen with the trinity desktop environment, which is very similar to KDE. To fix the problem, move the user's fontconfig-folder with this command in terminel:

`mv /home/user/.cache/fontconfig /home/user/.cache/fontconfig-old`

The folder gets re-created by the desktop-manager after next reboot. The system will run again.

**To Reproduce**
Described above...

**Expected behavior**
A program not rendering the system unbootable.

**Desktop (please complete the following information):**
 - PWA or Electron application
 - OS: Ubuntu 18.0.4 (Bionic)

**Additional context**

**Comment by 4gray** [comment]
Thank you for the bug report! 

You mean something like this https://github.com/sindresorhus/caprine/issues/1284, right? From what I understand it is an electron-related problem, will have to look for a solution.

Another question, did the problem appear in the new version (v0.10.0) or was it also in the previous one?

**Comment by mr-who2012** [comment]
Yes, it did look exactly like this! I was so shocked, the system didn't boot-up afterwards. This bug actually reveals a security-problem. I just ran the appimage-version of v0.10.0. No installation, nothing. It shouldn't be possible to mess-up the system in user-mode, like this. It took the whole morning to find and fix the problem. 

I didn't try earlier versions. Just stumbled upon this software and tried to watch TV with it. Turns out it's for IPTV only. I was actually looking for a good and quick TV-software for my DVB-C-card. I don't want to setup any servers for relaying it to some application. I just wanted a quick way to turn on my card's tuner. So...up to now vlc is the easiest way to go. However, your software looks better.

**Comment by 4gray** [comment]
Oh, I'm really sorry for that, need to investigate the issue in more detail. 
Just checked with Manjaro and i3 on my machine the font in the dialog looks corrupted too, but this has no effect on the system or other fonts. I will have to check if this was also the case in previous versions of the app.

**Comment by nemoskapetanos** [comment]
> Yes, it did look exactly like this! I was so shocked, the system didn't boot-up afterwards. This bug actually reveals a security-problem. I just ran the appimage-version of v0.10.0. No installation, nothing. It shouldn't be possible to mess-up the system in user-mode, like this. It took the whole morning to find and fix the problem. 
> 
> I didn't try earlier versions. Just stumbled upon this software and tried to watch TV with it. Turns out it's for IPTV only. I was actually looking for a good and quick TV-software for my DVB-C-card. I don't want to setup any servers for relaying it to some application. I just wanted a quick way to turn on my card's tuner. So...up to now vlc is the easiest way to go. However, your software looks better.

The simplest tuner card software out there is Kaffeine. It's available as a deb, app image & flatpak. Just install it and on the initial screen there's an option to setup DVB TV. Heaps of setup guides on Youtube. Good luck!
